meteor/client/ui/RundownView.tsx

```diff
@@ -1504,7 +1504,12 @@ export const RundownView = translateWithTracker<IProps, IState, ITrackedProps>((
 				playlistId,
 			})
 			this.autorun(() => {
-				let playlist = RundownPlaylists.findOne(playlistId)
+				let playlist = RundownPlaylists.findOne(playlistId, {
```

Comment:
To ensure type safety, do a Pick<RundownPlaylist, '_id' | 'studioId'> here

Reply:
cast to `Pick<RundownPlaylist, '_id' | 'studioId'> | undefined`.